### ISSUE TITLE
1.1 preview1, Generate wrong `xaml.g.h` file

### ISSUE THREAD

**Issue by shelllet** [microsoft/microsoft-ui-xaml#6916]
### Describe the bug

bad code， missing some codes and closing character.
![image](https://user-images.githubusercontent.com/16729017/160958794-bb0eae05-231d-493c-8d72-9d7ff75e145d.png)

right code:
![image](https://user-images.githubusercontent.com/16729017/160958853-4f4fbd49-2db4-4d20-a85a-da82fe0d6f03.png)

### Steps to reproduce the bug

write the flowing idl code:
```
namespace App1
{
	 unsealed runtimeclass MyPage : Microsoft.UI.Xaml.Controls.Page
	{
		String Hello;
	}

		[default_interface]
	runtimeclass BlankPage1 : MyPage
	{
		BlankPage1();
		Int32 MyProperty;
	}
}
```

### Expected behavior

compile failed.

### Screenshots

![image](https://user-images.githubusercontent.com/16729017/160959002-bc92a542-eee2-4dfc-a28b-3bc15fd76b15.png)

**Comment by asklar** [comment]
CC @RealTommyKlein @codendone

**Comment by RealTommyKlein** [comment]
@shelllet @asklar - thanks for the report, this will be fixed in 1.1 Preview 2.

**Comment by asklar** [comment]
Closing since we've taken a fix.
